debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java

```diff
@@ -61,27 +76,46 @@ public void testParsingInsert() throws Exception {
         assertThat(entry.getNewValues().get(5).getColumnData()).isNull();
         assertThat(entry.getNewValues().get(6).getColumnData()).isNull();
         assertThat(entry.getNewValues().get(7).getColumnData()).isNull();
+        assertThat(entry.getNewValues().get(8).getColumnData()).isNull();
```

Comment:
Why is it null? So we actually can *not* export those CLOB/BLOB values? If so, is it stated in the docs?

Reply:
The Oracle connector hasn't officially supported clob/blob data types ever.  The XStream `LcrEventHandler` requires the chunk processing methods to be implemented which never were and LogMiner requires that we support multi-entry interpretation of DML statements to get the clob/blob values.  These data types aren't mentioned in the Oracle docs at all presently.

Reply:
I guess what's putting me off is the title of this PR: "Fix Oracle LogMiner emitting Clob/Blob as nulls". I read this as if emitting null is the current behavior, and this PR is going to fix this. But as per what you say, that's the case actually? So what is this PR doing then actually? Ignoring CLOB/BLOB more gracefully?

Reply:
I agree that the commit message and title is definitely a bit misleading and I should probably change it to something more accurate to what we're trying to accomplish.  Let me try to explain and maybe we can decide on a better one.

In 1.4.0 and 1.4.1 while streaming, LogMiner was actually picking up CLOB data types for the reporter's use case and it was streaming the values without a problem.  With some changes we made to improve the DML parser, this lead to a situation where we stopped picking up the clob values when emitting events, hence the reference to _null_.  The overall goal was then to restore the behavior so that clob values were being re-emitted as the were in 1.4.0 and 1.4.1.

In addition, while testing, I noticed that clobs were being emitted in the snapshot but as `oracle.sql.CLOB@93948f1a` values rather than the actual character data in the column, hence the change to the value converter using `subString`. 
wdyt?